frontend/resourceadm/pages/DeployResourcePage/DeployResourcePage.tsx

```diff
@@ -223,31 +233,35 @@ export const DeployResourcePage = ({
 
         return (
           <>
-            <Heading size='large' spacing level={1}>
+            <StudioHeading size='lg' spacing level={1}>
               {t('resourceadm.deploy_title')}
-            </Heading>
+            </StudioHeading>
             <div className={classes.contentWrapper}>
               {displayStatusCard()}
-              <Paragraph size='small' className={classes.informationText}>
+              <StudioParagraph size='sm' className={classes.informationText}>
                 <Trans i18nKey='resourceadm.deploy_description'>
-                  <Link href='https://www.altinn.no/' rel='noopener noreferrer' target='_blank'>
+                  <StudioLink
+                    href='https://www.altinn.no/'
```

Comment:
Might be worth considering adding the different URLs to constants in a separate file? 😄